### ISSUE TITLE
Allow to configure addons more directly the web version

### ISSUE THREAD

**Issue by lwcorp** [Stremio/stremio-features#842]
**Stremio Version**
Web

**Is your feature request related to a problem? Please describe.**
If one clicks Configure in an addon, the result is not directly installable since stremio://[something.com/manifest.json is not clickable since you don't register the stremio:// protocol in the web version.

**Describe the solution you'd like**
Allow to configure more directly.

**Describe alternatives you've considered**
As [commented here](#issuecomment-2103119328) it's possible to paste the result inside "+Add addon", but it's really not intuitive.
Keep in mind the web is kind of a frontend for non Desktop versions, since in those versions it's very unintuitive to install addons without a mouse/keyboard, so it should be direct and to the point.

**Additional context**
![image](https://github.com/Stremio/stremio-features/assets/1773306/b42354ac-4e92-4b1d-b796-ae0c7fe9c4c0)
![image](https://github.com/Stremio/stremio-features/assets/1773306/1e7370ec-dfd1-4f35-8f86-226a179aa885)

**Comment by jaruba** [comment]
if you copy the stremio:// link, then go to the addons page in Stremio Web, there is a big green "+ Add Addon" button, press it, paste the link and press install

**Comment by lwcorp** [comment]
You're right, thanks! I've updated the issue to requesting this process to be much clearer.
